data_management/opensearch_indexer/opensearch_indexer/text_extraction.py

```diff
@@ -76,5 +80,56 @@ def extract_text(file_stream: bytes, file_extension: str) -> str:
     ) as temp:
         temp.write(file_stream)
         temp.flush()
-        context = textract.process(temp.name)
-    return context.decode("utf-8")
+        file_path = temp.name
+
+        try:
```

Comment:
adding a couple of tests for these scenarios inside of
`data_management/opensearch_indexer/tests/test_text_extraction.py` would be good